configs/sst_rhel_lightspeed.yaml

```
@@ -0,0 +1,24 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: RHEL Lightspeed
+  description: Command Line Assistant powered by RHEL Lightspeed.
+  maintainer: sst_rhel_lightspeed
+  packages:
+    - command-line-asssitant
+  package_placeholders:
+    - srpm_name: command-line-assistant
+      rpms:
+        - rpm_name: command-line-assistant
+          description: Command Line Assistant
+          dependencies:
+            - tomli
+  labels:
+    - c9s
+    - c10s
```

c9s is no longer tracked in Content Resolver, but if you expect to have this in RHEL 11, then ELN should be tracked instead (even as a placeholder):
```suggestion
  labels:
    - eln
    - c10s
```